src/main/java/com/cse/helium/app/utils/InvalidParameterResponses.java

```diff
@@ -0,0 +1,127 @@
+package com.cse.helium.app.utils;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Component;
+
+@Component
+public class InvalidParameterResponses {
+
+  private static final String VALIDATION_ERROR_TEMPLATE = "    {\n"
+      + "      \"code\": \"InvalidValue\",\n"
+      + "      \"target\": \"%s\",\n"
+      + "      \"message\": \"%s\",\n"
+      + "    }";
+
```

Comment:
Some of your error messages don't have the parameter names in camelCase. This won't cause webv to fail, but you might  want to update them, since the queries are case-sensitive.

Reply:
done